qiskit/providers/models/backendproperties.py

```diff
@@ -234,6 +235,8 @@ def from_dict(cls, data):
         backend_name = in_data.pop('backend_name')
         backend_version = in_data.pop('backend_version')
         last_update_date = in_data.pop('last_update_date')
+        if isinstance(last_update_date, str):
+            last_update_date = dateutil.parser.parse(last_update_date)
```

Comment:
Do you think this should probably be in the `__init__` instead of `from_dict`? (I'm not sure if it makes sense in `__int__` but it might)

Reply:
You're right, this is probably better to be in `__init__`. That would allows a provider to construct a `BackendProperties` without having to convert the datetime string, if that's the format they use.